author.php

```diff
@@ -9,13 +9,13 @@
  * @since    Timber 0.1
  */
 
-wp_register_script( 'author', get_template_directory_uri() . '/assets/js/author.js', [ 'jquery', 'main' ], '0.0.1', true );
+wp_register_script( 'author', get_template_directory_uri() . '/assets/js/load_more.js', [ 'jquery', 'main' ], '0.0.1', true );
```

Comment:
@pablocubico I missed that.. please change the handle string to 'load_more' (does not make any difference, just for consistency reasons)

Reply:
Done!